zip-0320.html

```diff
@@ -9,10 +9,10 @@
     <section>
         <pre>ZIP: 320
 Title: Defining an Address Type to which funds can only be sent from Transparent Addresses
-Owners: Daira-Emma Hopwood &lt;daira@electriccoin.co&gt;
+Owners: Daira-Emma Hopwood &lt;daira-emma@electriccoin.co&gt;
         Kris Nuttycombe &lt;kris@nutty.land&gt;
 Credits: Hanh
-Status: Draft
+Status: Proposed
```

Comment:
Is automatic rendering in `main` still not working? Bleh.

Reply:
It wasn't last time I looked, and I want to test it with smaller PRs than this one!